app/src/organisms/ProtocolSetupParameters/AnalysisFailedModal.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import { useHistory } from 'react-router-dom'
+import {
+  COLORS,
+  DIRECTION_COLUMN,
+  Flex,
+  SPACING,
+  TYPOGRAPHY,
+} from '@opentrons/components'
+
+import { StyledText } from '../../atoms/text'
+import { SmallButton } from '../../atoms/buttons'
+import { Modal } from '../../molecules/Modal'
+
+import type { ModalHeaderBaseProps } from '../../molecules/Modal/types'
+
+interface AnalysisFailedModalProps {
+  errors: string[]
+  protocolId: string
+  setShowAnalysisFailedModal: (showAnalysisFailedModal: boolean) => void
+}
+
+export function AnalysisFailedModal({
+  errors,
+  protocolId,
+  setShowAnalysisFailedModal,
+}: AnalysisFailedModalProps): JSX.Element {
+  const { t } = useTranslation('protocol_setup')
+  const history = useHistory()
+  const modalHeader: ModalHeaderBaseProps = {
+    title: t('protocol_analysis_failed'),
+    iconName: 'information',
+    iconColor: COLORS.black90,
+    hasExitIcon: true,
+  }
+
+  const handleRestartSetup = (): void => {
+    history.push(`/protocols/${protocolId}`)
+  }
+
+  return (
+    <Modal
+      header={modalHeader}
+      onOutsideClick={() => {
+        setShowAnalysisFailedModal(false)
+      }}
```

Comment:
hmmm what you added is to fix an eslint error right? If so, you can ignore this comment lol

```suggestion
      onOutsideClick={() => 
        setShowAnalysisFailedModal(false)
      }
```

Reply:
yeah, to remove warning.
i will remove `{}`

Reply:
ah ok, you can leave it in!